packages/ember-data/lib/adapters/rest_adapter.js

```diff
@@ -217,6 +217,24 @@ export default Adapter.extend({
   coalesceFindRequests: false,
 
   /**
+    By default the RESTAdapter will expect response data to be in json format. Override `responseDataType`
+    to specify a different type.
+
+    ```javascript
+    App.ApplicationAdapter = DS.RESTAdapter.extend({
+      responseDataType: 'jsonp'
+      }
+    });
+    ```
+
+    Response data types are limited to types accepted by the [jQuery library](http://api.jquery.com/jQuery.ajax/).
+
+    @property responseDataType
+    @type {string}
```

Comment:
Can you use a capital `String` here for consistency?
Also can you add `@default 'json'`?